src/style.css

```diff
@@ -1,3 +1,79 @@
 body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
   margin: 0;
 }
+
+:root {
+  --header-height: 60px;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  position: absolute;
+  padding: 0 50px;
+  box-sizing: border-box;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  height: var(--header-height);
+  line-height: var(--header-height);
+  text-align: center;
+  text-transform: uppercase;
```

Comment:
Try to remove height and line-height from nav and set line-height: 60px to nav__link and rerun tests.